wallet-server-ui/src/service/websocket.service.ts

```diff
@@ -16,7 +16,6 @@ import { ConfirmationDialogComponent } from '~app/dialog/confirmation/confirmati
 import { AddressService } from './address.service'
```

Comment:
Please remove changes to this file.

Reply:
Resolved by 24e0bf481e989e782838ebfeccbb1400a717ddae